docs/diagrams/PaymentModificationClassDiagram.puml

```diff
@@ -0,0 +1,25 @@
+@startuml
```

Comment:
The displayed diagram's arrows looks white below, could there be something wrong with styling?